flexget/utils/tools.py

```diff
@@ -426,3 +426,25 @@ def get_latest_flexget_version_number():
 
 def get_current_flexget_version():
     return flexget.__version__
+
+
+def data_size_to_bytes(size, base2=False):
+    """
+    Expands a data size into its bytes as an integer i.e. '1 MB' becomes 1000000.
+    If base2 is specified then assume bibytes so i.e. '1 MB' becomes 1048576
+    """
+    if (base2):
+        split = 1024 ** 2
+    else:
+        split = 1000 ** 2
+
+    if size.split()[1] == 'GB' or size.split()[1] == "GiB":
```

Comment:
I'd probably do something like `unit, amount = size.split()` once, instead of referring to size.split()[0] and [1] each time.